docs/devsite-help/support.md

```diff
@@ -0,0 +1,223 @@
+# Support
+
+The vast majority of the code in the Google Cloud Libraries is autogenerated. If
+you're running into problems, it *probably* isn't in that code (although that's
+always a possibility). The problems we see being reported usually fall into a
+few different buckets, described below.
+
+In general, after reading the advice below, if you have a support contract with
+Google, please create an issue in the
+[support console](https://cloud.google.com/support/) instead of filing on
+GitHub. This will ensure a timely response.
+## Authentication issues
+
+Authentication is a complex topic. We would advise reading the general
+[Google Cloud Authentication documentation](https://cloud.google.com/docs/authentication)
+first in order to determine what authentication strategy is most suitable for
+your application. After that, the[.NET Authentication documentation]
```

Comment:
Just checking if we need a space between the and [.

Reply:
Yup, thanks!